### ISSUE TITLE
About dialog opens after 2s delay

### ISSUE THREAD

**Issue by UrtsiSantsi** [NickvisionApps/Parabolic#853]
On 2024.10.0 'About' dialog opens after 2s delay, but  'Preferences' and 'Keyboard Shortcuts' opens instantaneously. Opening it multiple consecutive times don't decrease the delay.
Other applications don't have such problem.

**Comment by nlogozzo** [comment]
That's because Parabolic will spin up a `yt-dlp`, a `ffmpeg` and an `aria2c` process to get their version numbers for the troubleshooting information section.

Maybe I could move this to a background task on startup so the versions will be instantly available when getting the troubleshooting information.

**Comment by nlogozzo** [comment]
Yeah the problem with AdwAboutDialog is that all the information needs to be provided to the dialog before it is opened. There is no signal that says a certain section was opened and then load the information dynamically.
